modules/gateways/CoinGate/client.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+require_once __DIR__ . '/init.php';
+
+CoinGate\Client::setAppInfo('WHMCS Extension', '2.0.0');
```

Comment:
I have followed the similar pattern most other payment modules for whmcs use (including old version) because 3rd party module installations happen through the FTP upload.

I was also thinking to include SDK as a submodule to keep things a bit more organized, maybe this could be a more cleaner approach, e.g. [omise example](https://github.com/sfalpha/whmcs-gateway-omise/tree/main/modules/gateways/omise)
Let me know.

Reply:
ok , lets leave it like this